bioinf_tools/fasta_analisys_tool.py

```diff
@@ -0,0 +1,105 @@
+GC_RANGE = (0, 100)
+STANDART_LENGHT = (0, 2**32)
+THRESHOLD_VALUE = (0, 41)
+
+
+def fasta_analysis(
+    seqs: dict,
+    gc_bounds: tuple = GC_RANGE,
+    length_bounds: tuple = STANDART_LENGHT,
+    quality_threshold: tuple = THRESHOLD_VALUE,
+) -> dict:
+    """
+    Filters values in the fast dictionary according to specified parameters
+
+    Arguments:
+      - seqs (dict), conditions values
+
+      Return:
+      - dictionary with values filtered by parameters"""
+
+    gc_bounds, length_bounds, quality_threshold = check_conditions(
+        gc_bounds, length_bounds, quality_threshold
+    )
+    filtered_seqs = {}
+    for name in seqs.keys():
+        seq, quality_info = seqs.get(name)
```

Comment:
Можно в одну строку:
```suggestion
    for name, (seq, quality_info) in seqs.items():
```
И при получения значения по ключу обычно используют такую запись:
```python
some_dict[key]
```